src/main/java/de/hysky/skyblocker/utils/ItemUtils.java

```diff
@@ -203,16 +204,16 @@ public static PropertyMap propertyMapWithTexture(String textureValue) {
         return Codecs.GAME_PROFILE_PROPERTY_MAP.parse(JsonOps.INSTANCE, JsonParser.parseString("[{\"name\":\"textures\",\"value\":\"" + textureValue + "\"}]")).getOrThrow();
     }
 
-    public static String getHeadTexture(ItemStack stack) {
-        if (!stack.isOf(Items.PLAYER_HEAD) || !stack.contains(DataComponentTypes.PROFILE)) return "";
+    public static Optional<String> getHeadTexture(ItemStack stack) {
```

Comment:
I would switch this back to its previous functionality and add an `getHeadTextureOptional` method that links to the original so as to avoid creating a bunch of unnecessary optionals in the mob glow every frame.